Poker.BE/Poker.BE.Service/Services/AuthenticationService.cs

```diff
@@ -12,112 +12,112 @@
 
-            try
-            {
-                result.Output = userManager.LogOut(user);
-                result.User = user.GetHashCode();
-                result.Success = true;
-            }
-            catch (UserNotFoundException e)
-            {
-                result.Success = false;
-                result.ErrorMessage = e.Message;
-            }
+			User user;
+			if (!Users.TryGetValue(request.User, out user))
+			{
+				result.Success = false;
+				result.ErrorMessage = "User ID not found";
+			}
 
-            return result;
-        }
+			try
+			{
+				result.Output = userManager.LogOut(user);
+				result.User = user.GetHashCode();
+				result.Success = true;
+			}
+			catch (UserNotFoundException e)
+			{
+				result.Success = false;
+				result.ErrorMessage = e.Message;
+			}
 
-        public SignUpResult SignUp(SignUpRequest request)
-        {
-            var result = new SignUpResult();
+			return result;
+		}
 
-            try
-            {
-                User user = userManager.AddUser(request.UserName, request.Password, request.Deposit);
-                result.User = user.GetHashCode();
-                Users.Add(user.GetHashCode(), user);
-                result.Success = true;
-            }
-            catch (UserNameTakenException e)
-            {
-                result.Success = false;
-                result.ErrorMessage = e.Message;
-            }
-            catch (IncorrectPasswordException e)
-            {
-                result.Success = false;
-                result.ErrorMessage = e.Message;
-            }
-            catch (InvalidPasswordException e)
-            {
-                result.Success = false;
-                result.ErrorMessage = e.Message;
-            }
-            catch (InvalidDepositException e)
-            {
-                result.Success = false;
-                result.ErrorMessage = e.Message;
-            }
-            return result;
-        }
+		public SignUpResult SignUp(SignUpRequest request)
+		{
+			var result = new SignUpResult();
 
-        public void Clear()
-        {
-            userManager.Clear();
-        }
-        #endregion
-    }
+			try
+			{
+				User user = userManager.AddUser(request.UserName, request.Password, request.Deposit);
+				result.User = user.GetHashCode();
+				Users.Add(user.GetHashCode(), user);
+				result.Success = true;
+			}
+			catch (UserNameTakenException e)
+			{
+				result.Success = false;
+				result.ErrorMessage = e.Message;
+			}
+			catch (IncorrectPasswordException e)
+			{
+				result.Success = false;
+				result.ErrorMessage = e.Message;
+			}
+			catch (InvalidPasswordException e)
+			{
+				result.Success = false;
+				result.ErrorMessage = e.Message;
+			}
+			catch (InvalidDepositException e)
+			{
+				result.Success = false;
+				result.ErrorMessage = e.Message;
+			}
+			return result;
+		}
+
+		public void Clear()
+		{
+			userManager.Clear();
+		}
+		#endregion
+	}
```

Comment:
# ❗️ massive change - for nothing
i don't know what you have changed here because the `git diff` is massive. but it seems that you tried to auto-format and it didn't go well. please check you auto format settings of visual studio and return them to default!